content/docs/postgres/query-performance.md

```diff
@@ -0,0 +1,446 @@
+---
+title: Optimize Postgres query performance
+subtitle: Learn about strategies for optimizing Postgres query performance
+enableTableOfContents: true
+---
+
+Many factors can impact query performance in Postgres, ranging from insufficient indexing and database maintenance to poorly optimized queries or inadequate system resources. With such a wide range of factors, it can be difficult to know where to start. In this topic, we'll look at several strategies you can use to optimize query performance in Postgres.
+
+- [Gather statistics](#gather-statistics)
```

Comment:
Is there a loose or implied order in this list? Any opportunity to group into categories of some kind? Analysis or tuning, for example? 

Maybe a future improvement for this page would be to get some visual schematic of these operations, to help a reader see options at a glance.

Reply:
Categorized the strategies as suggested. We'll do a visual schematic with the next iteration. Good idea!